### ISSUE TITLE
Pick-based calibration is broken

### ISSUE THREAD

**Issue by joelvbernier** [HEXRD/hexrdgui#691]
@psavery -- looks like the pick-based cal is broken in current master.  This is the first time I've really tested it since you removed the middle click.  This was a mixed powder/laue case (from the demo).  Looks like misplaced parenthesis?  See below:
```
Traceback (most recent call last):
  File "/Users/joel/Documents/GitHub/hexrdgui/hexrd/ui/calibration/calibration_runner.py", line 137, in finish_line
    self.pick_next_line()
  File "/Users/joel/Documents/GitHub/hexrdgui/hexrd/ui/calibration/calibration_runner.py", line 101, in pick_next_line
    self.finish()
  File "/Users/joel/Documents/GitHub/hexrdgui/hexrd/ui/calibration/calibration_runner.py", line 212, in finish
    self.run_calibration()
  File "/Users/joel/Documents/GitHub/hexrdgui/hexrd/ui/calibration/calibration_runner.py", line 221, in run_calibration
    instr_calibrator = run_calibration(picks, instr, materials)
  File "/Users/joel/Documents/GitHub/hexrdgui/hexrd/ui/calibration/pick_based_calibration.py", line 453, in run_calibration
    instr_calibrator = CompositeCalibration(instr, picks)
  File "/Users/joel/Documents/GitHub/hexrdgui/hexrd/ui/calibration/pick_based_calibration.py", line 395, in __init__
    calib = LaueCalibrator(
  File "/Users/joel/Documents/GitHub/hexrdgui/hexrd/ui/calibration/pick_based_calibration.py", line 86, in __init__
    % len(self._full_params, len(flags))
TypeError: len() takes exactly one argument (2 given)
```

**Comment by cjh1** [comment]
@joelvbernier I have fixed this up in #692

**Comment by psavery** [comment]
The assertion failing might be a problem that I'll need to look into
